backend/core/validations/user_validation/UserPersonalDataValidation.js

```diff
@@ -0,0 +1,11 @@
+const namesValidation = require('./NameValidation.js');
+const dateValidator = require('./DateValidator.js');
+
+function validateUserPersonalData(userPersonalData) {
```

Comment:
The file is in user_validation folder, I think you can get rid of User in the validateUserPersonalData.

Reply:
Has this been addressed?